webpack-common.js

```diff
@@ -61,46 +48,25 @@ export function getRules({ babelOptions, bundleStylesWithJs = false } = {}) {
   } else {
     // In production, we create a separate CSS bundle rather than include
     // styles with the JS bundle. This lets the style bundle load in parallel.
-
-    const cssLoaderOptions = {
-      importLoaders: 2,
-      sourceMap: true,
-      minimize: true,
-    };
-
     styleRules = [
       {
-        test: /\.css$/,
+        test: /\.(sc|c)ss$/,
         loader: ExtractTextPlugin.extract({
           fallback: 'style-loader',
           use: [
             {
               loader: 'css-loader',
-              options: cssLoaderOptions,
-            },
-            {
-              loader: 'postcss-loader',
               options: {
-                plugins: postCssPlugins,
+                importLoaders: 2,
                 sourceMap: true,
+                minimize: true,
               },
             },
-          ],
-        }),
-      },
-      {
-        test: /\.scss$/,
-        loader: ExtractTextPlugin.extract({
-          fallback: 'style-loader',
-          use: [
-            {
-              loader: 'css-loader',
-              options: cssLoaderOptions,
-            },
             {
               loader: 'postcss-loader',
               options: {
                 plugins: postCssPlugins,
+                sourceMap: true,
```

Comment:
I basically merged the two sets of rules into a single one (instead of having two regular expressions, we only have one, which is fine).